.travis.yml

```diff
@@ -0,0 +1,8 @@
+language: python
+python:
+  - "2.7"
+cache: pip
+install:
+  - pip install -r requirements.txt
+script:
+  - python test.py
```

Comment:
Because it will build for a `inifinite` number of times watching file changes and will not pass for now.
Have a look on its output:

```shell
Worker information
hostname: i-0f6ba05-precise-production-2-worker-org-docker.travisci.net:feb3e67c-f78e-4ecd-94d9-0ba3bb8edbee
version: v2.5.0 https://github.com/travis-ci/worker/tree/da3a43228dffc0fcca5a46569ca786b22991979f
instance: 48066c3:travis:python
startup: 765.298551ms

Skipping a long starting log......................

  Running setup.py install for watchdog
    Installing watchmedo script to /home/travis/virtualenv/python2.7.9/bin
  Compiling /tmp/pip-build-xZvIXJ/jinja2/jinja2/asyncfilters.py
  
  Compiling /tmp/pip-build-xZvIXJ/jinja2/jinja2/asyncsupport.py
  
  Running setup.py install for easywatch
  Running setup.py install for docopt
  Running setup.py install for staticjinja
    Installing staticjinja script to /home/travis/virtualenv/python2.7.9/bin
Successfully installed MarkupSafe-1.0 PyYAML-3.12 argh-0.26.2 docopt-0.6.2 easywatch-0.0.3 jinja2-2.9.6 pathtools-0.1.2 staticjinja-0.3.3 watchdog-0.8.3
$ python build.py
Rendering coc.html...
Rendering faqs.html...
Rendering gallery.html...
Rendering index.html...
Rendering sponsors.html...
Rendering sponsorship.html...
Rendering volunteer.html...
Watching '/home/travis/build/pythonindia/inpycon2017/templates' for changes...
Press Ctrl+C to stop.
```